connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -317,14 +318,26 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
             extracted.length > 0 &&
             extracted.length <= MAX_SMALL_DOCUMENT_TXT_LEN
           ) {
+            const formattedDocumentContent = formatDocumentContent({
+              title: pageTitle,
+              content: extracted,
+              url: request.url,
+            });
+            if (!formattedDocumentContent) {
+              childLogger.info(
+                {
+                  documentId,
+                  configId: webCrawlerConfig.id,
+                  url,
+                },
+                `Invalid url. Skipping`
```

Comment:
nit: I would align the log with the method called. `Invalid document or URL, skipping`